src/VisualStudio/Core/Def/HACK_ThemeColorFixer.cs

```diff
@@ -108,6 +110,10 @@ public void RefreshThemeColors()
             UpdateForegroundColor(ClassificationTypeNames.ExtensionMethodName, sourceFormatMap, targetFormatMap);
             UpdateForegroundColor(ClassificationTypeNames.PropertyName, sourceFormatMap, targetFormatMap);
             UpdateForegroundColor(ClassificationTypeNames.EventName, sourceFormatMap, targetFormatMap);
+            UpdateForegroundColor(ClassificationTypeNames.NamespaceName, sourceFormatMap, targetFormatMap);
+            UpdateForegroundColor(ClassificationTypeNames.LabelName, sourceFormatMap, targetFormatMap);
+
+            UpdateForegroundColor(ClassificationTypeNames.StaticSymbol, sourceFormatMap, targetFormatMap);
```

Comment:
i thought thse didn't have colors?  or is this just for completeness?

Reply:
It is leftover from before then Foreground and Background options were disabled.  I'll remove. Thanks